data/formats-data.js

```diff
@@ -4832,7 +4839,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	zorua: {
-		randomBattleMoves: ["suckerpunch", "extrasensory", "darkpulse", "hiddenpowerfighting", "uturn", "knockoff"],
+		randomBattleMoves: ["suckerpunch", "extrasensory", "darkpulse", "nightdaze", "uturn", "knockoff", "nastyplot", "swordsdance"],
```

Comment:
hp fighting hits pawniard, you should keep it for np

Reply:
i cant keep something for one pokemon in rands

Reply:
hp fighting goes well with np - so if you roll np, you'd get night daze/dpulse, esp, hp fighting, otherwise, sd rolls knockoff, sucker, uturn

Reply:
the relevant setup move only needs two moves of the relevant category, and no-setup sets are possible, but I see the dark coverage still being useful so ok